src/internal/helm/repo.go

```diff
@@ -15,6 +15,25 @@ import (
 	"helm.sh/helm/v3/pkg/repo"
 )
 
+// CreateChartFromLocalFiles creates a chart archive from a path to a chart on the host os
+func CreateChartFromLocalFiles(chart types.ZarfChart, destination string) string {
```

Comment:
Is there any value in running a high-level [validation](https://github.com/helm/helm/blob/v3.10.0/pkg/chart/chart.go#L124) of the chart prior to package? Maybe this is covered by the processes below and failing to create an archive if not a compliant chart.

Reply:
My thinking here is to fail early and fail fast. Packaging should attempt (in my opinion) to validate some aspects of what is going to be deployed such that time isn't wasted transferring that artifact to the target environment and finding out that it is broken. I may be incorrect in assuming Zarf wants to do this though.

Reply:
Zarf has a validation package that runs during package create for this purpose but mainly focuses on zarf isms right now, I have 0 complaints about making that more robust to avoid finding out in air gap something is a problem.

Reply:
Added a validation check to all functions that load helm charts